gta-gallery/src/components/header/index.jsx

```diff
@@ -47,16 +46,20 @@ export const Header = () => {
   };
 
   const userInitials = useMemo(() => {
-    if (isEmpty(currentUser)) return;
-    return `${currentUser.firstName.charAt(0)} ${currentUser.lastName.charAt(
-      0
-    )}`;
-  }, [currentUser]);
+    let result = '';
+    if (firstName) {
+      result += firstName.charAt(0) + ' ';
+    }
+    if (lastName) {
+      result += lastName.charAt(0);
+    }
```

Comment:
Do we have both firstName and lastName as mandatory fields? If so we can just check if either of them don't exist and just return.